pyvast/pyvast/vast.py

```diff
@@ -19,12 +19,13 @@
 
 import asyncio
 import logging
+from distutils.spawn import find_executable
 
 
 class VAST:
     """A VAST node handle"""
 
-    def __init__(self, binary="vast", endpoint=None, logger=None):
+    def __init__(self, binary="vast", endpoint=None, container={"runtime":"docker", "name":"vast"}, logger=None):
```

Comment:
IIUC this runs `docker exec vast`. Should it instead run `docker run tenzir/vast:v2.0.0`, defaulting to the current version?

Reply:
Hello!

In here assumption is that there is already a containerized VAST  instance ingesting logs. `docker run tenzir/vast:v2.0.0`  would create a new container without any data. That is also the reason "name" key in `{"runtime":"docker", "name":"vast"}` is the container-name. A future enhancement on this could be auto-discovery of VAST container while instantiating pyvast.

Regards.

Reply:
I understand your intent now. I'll push a small change to the README directly onto this PR and then merge it. Thanks for your contribution! 🚀

Reply:
@KaanSK can you check the box in this PR that allows maintainers to push to your PR directly, so I can add a changelog entry and do some wordsmithing on the PyVAST README?

Reply:
I thought it was enabled by default but I approved edits by maintainers again.

Reply:
It won't let me push. Probably something on my end. Mind applying the diff?

Reply:
Diff applied. Hope it works now:).